src/rnp/rnp.c

```diff
@@ -393,24 +381,13 @@ rnp_cmd(rnp_cfg_t *cfg, rnp_t *rnp, int cmd, char *f)
                                 rnp_cfg_getbool(cfg, CFG_DETACHED)) == RNP_OK;
         }
         break;
-    case CMD_DECRYPT: {
-        if (f) {
-            const char *out = rnp_cfg_get(cfg, CFG_OUTFILE);
-            repgp_set_input(io, create_filepath_handle(f));
-            repgp_set_output(io, create_filepath_handle(out));
-        } else {
-            repgp_set_input(io, create_stdin_handle());
-            repgp_set_output(io,
-                             create_buffer_handle((size_t) rnp_cfg_getint(cfg, CFG_MAXALLOC)));
-        }
-        ret = (RNP_SUCCESS == repgp_decrypt(&ctx, io));
```

Comment:
I would rather see ``rnp_process_stream`` called by ``repgp_decrypt``.  Also implementation from ``init_stdin_src`` needs to be moved to ``create_stdin_handle`` as it's obviously duplicated now.

Reply:
@ni4 what's your plan regarding `librepgp` API in the streamed approach?

Reply:
@ronaldtse @flowher I think at first I will finish all further stream-based code and then we'll decide how to better glue it with repgp. Actually, it will be just one call for data parsing - rnp_process_stream, outputting decrypted/verified data and giving other information (like signatures) via the callback.

Reply:
👍